src/main/java/no/sikt/nva/scrapers/LicenseScraper.java

```diff
@@ -78,7 +78,7 @@ private boolean isNotAVersion(String lastPathElement) {
 
     private String trim(String licenseUri) {
         return Optional.of(licenseUri.replaceAll("[\\p{Cf}]", ""))
-                   .map(s -> s.replace("\u200B", StringUtils.EMPTY_STRING)).get();
+                   .map(s -> s.replaceAll("(\\n)|(\\s)|(\u200b)", StringUtils.EMPTY_STRING)).get();
```

Comment:
Regex for removing empty spaces and linebreaks for license uri

Reply:
call the constant for MATCH_WHITE_SPACE or something similar.